.gitignore

```diff
@@ -111,3 +111,8 @@ ENV/
 
 # Profiling results
 profiling/*.prof
+
+.idea/
+test.py
```

Comment:
This is useful for development. But we need to discuss and document this properly.

Reply:
I have documented this.